data/habitat_metrics_table_GH.csv

```diff
@@ -0,0 +1 @@
SmetaTopD_mean,wae,less oxy-thermal habitat,epilimnion meters,mean epilmnion meters during stratified period,,,linear distance (m)
SmetaBotD_mean,wae,less oxy-thermal habitat,hypolimnion meters,mean hypolimnion meters during stratified period,,,linear distance (m)
coef_var_30_60,wae,match-mismatch with yoy returning to littoral zone,variable temperatures post-ice-off,CV of surface temperature from 30-60 days post ice off.,"Sterns 1982, Hansen et al. 1998",,unitless (CV)
coef_var_0_30,wae,sudden temperature changes affect incubating eggs,variable temperatures post-ice-off,CV of surface temperature from 0-30 days post ice-off,Koonce et al. 1977,,unitless (CV)
mean_epi_hypo_ratio,wae,less oxy-thermal habitat,ratio of epilimnion to hypolimnion,mean epilmnion volume/mean hypolimnion volume,,,unitless (ratio)
mean_epi_vol,wae,less oxy-thermal habitat,epilimnion volume,mean epilmnion volume during stratified period,,,volume (m^3*1000)
mean_hyp_vol,wae,less oxy-thermal habitat,hypolimnion volume,mean hypolimnion volume during stratified period,,,volume (m^3*1000)
simulation_length_days,NA,NA,Numer of days simulated as open-water,Number of simulated days. Based on start/end being estimated ice on/off. ,,,days
volume_mean_m_3,NA,NA,Average volume of lake,Average volume of lake,,,m^3
volume_sum_m_3_day,NA,NA,Daily sum of lake volume,Daily sum of lake volume. Useful for standardizing Volume-based habitat metrics,,,day * volume (m^3)
GDD_wtr_10c,wae,growth rate,Growing Degree Days,Surface water Growing Degree Days with a 10 deg C base,Venturelli et al 2010,,Day * degree (C )
GDD_wtr_5c,wae,growth rate,Growing Degree Days,Surface water Growing Degree Days with a 5 deg C base,Venturelli et al 2010,,Day * degree (C )
optic_hab_8_64,wae,foraging efficacy,Optimal optical habitat area of benthos,Area of lake bottom where light falls between 8-64 lux ,Lester et al 2004,,Day * m^2
thermal_hab_11_25,wae,foraging efficacy,Optimal thermal habitat area of benthos,Area of lake bottom where temperature falls between 11-25 degrees C,Lester et al 2004,,Day * m^2
optic_thermal_hab,wae,foraging efficacy,Combined optimal thermal and optical habitat area of benthos,Area of lake bottom where both light falls between 8-64 lux AND temperature falls between 11-25 degrees C,Lester et al 2004,,Day * m^2
optic_hab_8_64_surf,wae,foraging efficacy,Optimal optical habitat area of lake,Relative surface area of lake where light falls between 8-64 lux ,Lester et al 2004,,Day * m^2
thermal_hab_11_25_surf,wae,foraging efficacy,Optimal thermal habitat area of lake,Relative surface area of lake where temperature falls between 11-25 degrees C,Lester et al 2004,,Day * m^2
optic_thermal_hab_surf,wae,foraging efficacy,Combined optimal thermal and optical habitat area of lake,Relative surface area of lake where both light falls between 8-64 lux AND temperature falls between 11-25 degrees C,Lester et al 2004,,Day * m^2
```

Comment:
Seems fine to open up using different methods. Want me to re-write it @lawinslow ?